devito/ir/equations/equation.py

```diff
@@ -73,6 +73,11 @@ def apply(self, func):
         kwargs['conditionals'] = {k: func(v) for k, v in self.conditionals.items()}
         return self.func(*args, **kwargs)
 
+    def __repr__(self):
+        if not self.is_Reduction:
+            return super().__repr__()
+        else:
+            return '%s = %s(%s, %s)' % (self.lhs, self.operation, self.lhs, self.rhs)
```

Comment:
Nitpicking, aesthetics, feel free to ignore...:

```
elif self.operation is OpInc:
    return 'Inc(%s, %s)' % ...
else:
    return '%s(%s, %s)' % (self.operation, ...)
```

Reply:
Ok. It's not really "used" anywhere was just easier for debug with `embed`, will update is nicer indeed